utils/docker/Dockerfile.el.8

```
@@ -151,8 +149,17 @@ RUN if [ "x$BULLSEYE" != "x" ]; then \
 
 USER daos_server:daos_server
 
-# Install scons from pip to avoid dependency cycle bug.
-RUN python3 -m pip --no-cache-dir --disable-pip-version-check install --user scons
+# Setup a python venv so that python packages can be installed locally.
+RUN python3 -m venv /home/daos/venv
+ENV PATH=/home/daos/venv/bin:$PATH
+ENV VIRTUAL_ENV=/home/daos/venv/
```

Does this syntax work?
```suggestion
ENV VIRTUAL_ENV=/home/daos/venv/
RUN python3 -m venv $VIRTUAL_ENV
ENV PATH=$VIRTUAL_ENV/bin:$PATH
```

---

That would mean that VIRTUAL_ENV was set for the python process which I was trying to avoid.